content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "Production environment"
+description: Create a production-quality Kubernetes cluster
+weight: 30
+no_list: true
+---
+<!-- overview -->
+
+A production-quality Kubernetes cluster requires planning and preparation.
+If co-workers and customers depend on Kubernetes being available to run critical
+workloads, it is your job to make sure that the platform is set up and
+maintained to meet their demands.
```

Comment:
```suggestion
If Kubernetes runs critical workloads, Kubernetes must be configured to be resilient.
```
I tried to provide a clear, concise alternative.

Reply:
Okay. I used what you said and reworded slightly. Thanks.